cli/fathom_web/utils.py

```diff
@@ -17,26 +17,45 @@ def tensor(some_list):
     return torch.tensor(some_list, dtype=torch.float)
 
 
-def tensors_from(pages, shuffle=False):
+def tensors_from(vector_data, shuffle=False, exclude=None):
     """Return (inputs, correct outputs, number of tags that are recognition
     targets) tuple.
 
     Can also shuffle to improve training performance.
 
+    :arg exclude: List of rules to omit
+
     """
+    if exclude is None:
+        exclude = []
+    excluded_indices = [vector_data['header']['featureNames'].index(e) for e in exclude]
+
     xs = []
     ys = []
     num_targets = 0
+    pages = vector_data['pages']
```

Comment:
I really like this little change.

Reply:
Thanks! I had little choice; I needed to start passing the whole vector file in here instead of just the pages, and I wasn't about to say `vector_data['pages']` 3 times. :-)